### ISSUE TITLE
Code is generated to load whitespace

### ISSUE THREAD

**Issue by 0xR** [aruntk/wc-loader#16]
Using the [webpack analyze tool](https://webpack.github.io/analyse/#module/80) I discovered that sometimes code like this is generated:

```js
/*__wc__loader*/

(function (document) {
  var _htmlStr = "\n\n\n\n\n\n\n\n\n\n\n\n\n";
  if (document.head) {
    var el = document.head;
    var div = document.createElement('div');
    div.innerHTML = _htmlStr;
    while (div.children.length > 0) {
      el.appendChild(div.children[0]);
    }
  } else {
    document.write(_htmlStr);
  }
})(document);

(function (document) {
  var _htmlStr = "<dom-module ...
```

This seems unnecessary and not doing this will improve runtime performance and bundle size.

**Comment by aruntk** [comment]
Fixed in 94a50d1

For html minify option refer 

https://github.com/aruntk/wc-loader/blob/master/README.md#minify-html

**Comment by 0xR** [comment]
That has fixed it, thanks!

Also thanks for adding the html-minify option. I was looking for that earlier and didn't find it. As a workaround I was using `html-minify-loader` before applying `wc-loader`.
